<launch>
    <node name="left_motor_controller" pkg="motor_controller" type="motor_controller">
        <param name="driver_node_name" value="left_motor_driver"/>
        <param name="outlier_gap_bound" value="1.0"/>
        <param name="kp" value="1"/>
        <param name="ki" value="0"/>
        <param name="kd" value="0"/>
        <param name="reverse_rotation" value="false"/>
        <param name="goal_topic_name" value="left_motor_goal"/>
        <param name="radius" value="0.041"/><!--meter-->
    </node>
    <node name="right_motor_controller" pkg="motor_controller" type="motor_controller">
        <param name="driver_node_name" value="right_motor_driver"/>
        <param name="outlier_gap_bound" value="0.5"/>
        <param name="kp" value="1"/>
        <param name="ki" value="0"/>
        <param name="kd" value="0"/>
        <param name="goal_topic_name" value="right_motor_goal"/>
        <param name="reverse_rotation" value="true"/>
        <param name="radius" value="0.041"/>
    </node>
</launch>